### ISSUE TITLE
Add hardware acceleration guide

### ISSUE THREAD

**Issue by FlysoftBeta** [Fmstrat/winapps#370]
Although it is generally impossible to use vGPU on NVIDIA customer GPUs, we can leverage Intel's GVT-g to meet daily usage as it provides a practical virtual GPU. It would be nice to add hardware acceleration guide for Intel iGPUs.

**Comment by KernelGhost** [comment]
As far as I recall, Intel GVT-g supports only Intel Broadwell (5th generation) to Comet Lake (10th generation) processors. I tried to implement this on my system but was ultimately unsuccessful. Although I managed to create the vGPU and pass it to the Windows VM, I encountered a significant issue with what seemed to be a frame buffer width mismatch. I have a preliminary guide I wrote for setting up this feature, which I'm happy to share if you're interested. Perhaps you could identify the problem and suggest a solution. If we can get this feature working, I'd be glad to include it in the updated WinApps project documentation at https://github.com/winapps-org/winapps. Thanks!

**Comment by IPlayZed** [comment]
@KernelGhost I do not think that (due to vendor support from all 3 GPU makers) _most_ consumer cards would be ever feasible. I think there are two different directions the project could (if not already) go on:

1. When setting up the VM try to install all [VirtIO drivers](https://pve.proxmox.com/wiki/Windows_VirtIO_Drivers). If user wants to, offer allocating a block device and when setting up the VM in the background inject the VirtIO disk drivers. While these are not in itself a solution for HW acceleration, if well configured it could make most apps responsive enough. Maybe the project already does this though?
2. Support guiding through (or at least document in a project specific way) a script for setting up GPU pass-through. The process is pretty well-defined on many forums like the Arch Wiki and a lot more, so I will not link all, but if you also pair it with [Looking Glass](https://looking-glass.io), then you could drop RDP. This would be the cleanest, but might require some patches for Looking Glass upstream (for instance, I am not sure if it supports taking only a specific window as output in the guest). Again, most configuration with dual GPUs (unless muxed (see: [
XDC 2024 - Day 1 - October 9, 2024: Nvidia's part around 8:15:00](https://www.youtube.com/live/d0MP9-hFUZE?si=2QXkJnUeBmEa4Aui&t=29887) like old laptops) would work just fine. The script just needs the logic the explore the system's setup.
3. The list of hardware where vGPU splitting that OP or you outlined is pretty well-known. Similar to point 2. support a set-up through scripting (or at least document in a project specific way) if a supported hardware is identified.

Of course none of these would be enough for anything that does VM or such checks. It is very likely there is some nonsense (and I do not mean on-line games, those kind of have a legitimate reason to) program that checks. You just have to deal with that...
